packages/tempus/src/Tempus_Stepper_decl.hpp

```diff
@@ -167,7 +167,7 @@ class Stepper
 
   std::string stepperType_;             //< Name of stepper type
   bool useFSAL_ = false;                //< Use First-Step-As-Last (FSAL) principle
-  std::string ICConsistency_ = "None";  //< Type of consistency to apply to ICs.
```

Comment:
Should I patch up the latest PR also?

Reply:
I committed this change to the https://github.com/trilinos/Trilinos/tree/Tempus-Add-Initialization-to-Steppers-5908 branch, so you should "pull" the latest commit into your current changes.